risc0/zkvm/src/receipt.rs

```diff
@@ -134,6 +134,9 @@ pub struct SegmentReceipt {
     /// [SegmentReceipt::verify]) to cryptographically prove that this Segment
     /// was faithfully executed.
     pub seal: Vec<u32>,
+
+    /// Segment index within the [SessionReceipt]
+    pub index: u64,
```

Comment:
4 billion segments seems like a lot :)

Reply:
I like to be optimistic hahaha, but u32 is more portable. Fixed in: c041649754ba2478d005ad8b77cd01fe1c8102d5